index.d.ts

```diff
@@ -6,6 +6,13 @@ declare namespace camelcase {
 		@default false
 		*/
 		readonly pascalCase?: boolean;
+
+		/**
+		From `String.prototype.toLocaleUpperCase()`: The locale parameter indicates the locale to be used to convert to upper/lower case according to any locale-specific case mappings. If multiple locales are given in an Array, the best available locale is used. The default locale is the host environment’s current locale.
+
+		@default The host environment’s current locale
```

Comment:
This is incorrect syntax. `@default` only works with a JS value.

Reply:
Then I append following to the description of `locale` parameter:
"If not provided, host environment's current locale is used". 

Do you have other suggestion for such a case?

Reply:
You can just write `Default: The host environment’s current locale.`

Reply:
done.